src/PrestaShopBundle/Controller/Admin/Sell/Catalog/ManufacturerController.php

```diff
@@ -419,6 +421,46 @@ public function exportAction(ManufacturerFilters $filters)
             ;
     }
 
+    /**
+     * Deletes manufacturer logo image.
+     *
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
```

Comment:
Using array is deprecated since sf 4.*
```suggestion
     *     "is_granted('update', request.get('_legacy_controller'))",
```

Reply:
Fixed